src/main/java/org/folio/rest/delegate/StreamingProcessDelegate.java

```diff
@@ -0,0 +1,62 @@
+package org.folio.rest.delegate;
+
+import javax.script.ScriptException;
+
+import org.camunda.bpm.engine.delegate.DelegateExecution;
+import org.camunda.bpm.engine.delegate.Expression;
+import org.camunda.bpm.model.bpmn.instance.FlowElement;
+import org.folio.rest.service.ScriptEngineService;
+import org.folio.rest.service.StreamService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Scope;
+import org.springframework.stereotype.Service;
+
+@Service
+@Scope("prototype")
+public class StreamingProcessDelegate extends AbstractRuntimeDelegate {
+
+  @Autowired
+  private ScriptEngineService scriptEngineService;
+
+  @Autowired
+  private StreamService streamService;
+
+  private Expression script;
+
+  private Expression scriptType;
+
+  @Override
+  public void execute(DelegateExecution execution) throws Exception {
+    FlowElement bpmnModelElement = execution.getBpmnModelElementInstance();
+    String delegateName = bpmnModelElement.getName();
+
+    if (scriptType != null && script != null) {
+      String scriptTypeValue = scriptType.getValue(execution).toString();
+
+      scriptEngineService.registerScript(scriptTypeValue, delegateName, script.getValue(execution).toString());
+
+      String primaryStreamId = (String) execution.getVariable("primaryStreamId");
+
+      log.info(String.format("%s STARTED", delegateName));
+      streamService.getFlux(primaryStreamId).map(d -> {
```

Comment:
This is the instance where we have the use StreamService.map() not Flux.map()

Nothing is done with the Flux that is returned by the map in this case.